main_api.py

```diff
@@ -38,6 +38,8 @@
 token_secret_key = os.getenv("SECRET_KEY", "secret")
 ALGORITHM = "HS256"
 ENCODING = "utf-8"
+LOGIN_TOKEN_EXPIRE_DAYS = 1
```

Comment:
I think Ibrahims expectation was that a login token (more properly called an auth token), should only last a few hours.

Reply:
Done.